gatk-variantcalling.wdl

```diff
@@ -248,7 +249,17 @@ workflow GatkVariantCalling {
             dockerImage = dockerImages["picard"]
     }
 
+    call vt.Normalize as normalize {
+        input:
+            inputVCF = gatherVcfs.outputVcf,
+            inputVCFIndex = gatherVcfs.outputVcfIndex,
+            referenceFasta = referenceFasta,
+            referenceFastaFai = referenceFastaFai,
+            outputPath = outputDir + "/" + vcfBasename + ".normalized_decomposed.vcf.gz",
+    }
+
     output {
+        File outputVcfNormalizedDecomposed = normalize.outputVcf
```

Comment:
I'm wondering if this should replace `outputVcf` instead of being a new output. You would probably want to use the normalized VCF for all downstream usages anyway. 

Either way, there should be an index added for this VCF file.

Reply:
Yeah, you are right. I will replace it right away.

Reply:
I have added samtools tabix function